Makefile

```diff
@@ -39,64 +39,140 @@ allcheck:
+	@printf "$$(tput bold)* CI-Netlify: Done! *$$(tput sgr0)\n"
+	@printf "$$(tput bold)*********************$$(tput sgr0)\n"
+
+
+## Actual Rules (Travis)
+
+.PHONY: ci-formatting
+ci-formatting:
 	@printf "$$(tput bold)******************$$(tput sgr0)\n"
 	@printf "$$(tput bold)* CI: Formatting *$$(tput sgr0)\n"
 	@printf "$$(tput bold)******************$$(tput sgr0)\n"
 	@CI=true ./tools/run_cargo_fmt.sh diff
 	@./tools/check_wildcard_imports.sh
+
+.PHONY: ci-tools
+ci-tools:
 	@printf "$$(tput bold)*************$$(tput sgr0)\n"
 	@printf "$$(tput bold)* CI: Tools *$$(tput sgr0)\n"
 	@printf "$$(tput bold)*************$$(tput sgr0)\n"
 	@for f in `./tools/list_tools.sh`; do echo "$$(tput bold)Build & Test $$f"; cd tools/$$f && CI=true RUSTFLAGS="-D warnings" cargo build --all-targets || exit 1; cd - > /dev/null; done
+
+.PHONY: ci-libraries
+ci-libraries:
 	@printf "$$(tput bold)*****************$$(tput sgr0)\n"
 	@printf "$$(tput bold)* CI: Libraries *$$(tput sgr0)\n"
 	@printf "$$(tput bold)*****************$$(tput sgr0)\n"
 	@cd libraries/tock-cells && CI=true cargo test
 	@cd libraries/tock-register-interface && CI=true cargo test
+
+.PHONY: ci-archs
+ci-archs:
 	@printf "$$(tput bold)*************$$(tput sgr0)\n"
 	@printf "$$(tput bold)* CI: Archs *$$(tput sgr0)\n"
 	@printf "$$(tput bold)*************$$(tput sgr0)\n"
 	@for f in `./tools/list_archs.sh`; do echo "$$(tput bold)Test $$f"; cd arch/$$f; CI=true TOCK_KERNEL_VERSION=ci_test cargo test || exit 1; cd ../..; done
+
+.PHONY: ci-chips
+ci-chips:
 	@printf "$$(tput bold)*************$$(tput sgr0)\n"
 	@printf "$$(tput bold)* CI: Chips *$$(tput sgr0)\n"
 	@printf "$$(tput bold)*************$$(tput sgr0)\n"
 	@for f in `./tools/list_chips.sh`; do echo "$$(tput bold)Test $$f"; cd chips/$$f; CI=true TOCK_KERNEL_VERSION=ci_test cargo test || exit 1; cd ../..; done
-	@printf "$$(tput bold)**************$$(tput sgr0)\n"
-	@printf "$$(tput bold)* CI: Syntax *$$(tput sgr0)\n"
-	@printf "$$(tput bold)**************$$(tput sgr0)\n"
-	@CI=true $(MAKE) allcheck
+
+.PHONY: ci-kernel
+ci-kernel:
 	@printf "$$(tput bold)**************$$(tput sgr0)\n"
 	@printf "$$(tput bold)* CI: Kernel *$$(tput sgr0)\n"
 	@printf "$$(tput bold)**************$$(tput sgr0)\n"
 	@cd kernel && CI=true TOCK_KERNEL_VERSION=ci_test cargo test
+
+.PHONY: ci-syntax
+ci-syntax:
+	@printf "$$(tput bold)**************$$(tput sgr0)\n"
+	@printf "$$(tput bold)* CI: Syntax *$$(tput sgr0)\n"
+	@printf "$$(tput bold)**************$$(tput sgr0)\n"
+	@CI=true $(MAKE) allcheck
+
+.PHONY: ci-compilation
+ci-compilation:
 	@printf "$$(tput bold)*******************$$(tput sgr0)\n"
 	@printf "$$(tput bold)* CI: Compilation *$$(tput sgr0)\n"
 	@printf "$$(tput bold)*******************$$(tput sgr0)\n"
 	@CI=true $(MAKE) allboards
+
+.PHONY: ci-special-targets
+ci-special-targets:
 	@printf "$$(tput bold)***********************$$(tput sgr0)\n"
 	@printf "$$(tput bold)* CI: Special Targets *$$(tput sgr0)\n"
```

Comment:
Could we rename this one to something more descriptive? I tend to keep asking myself what those special targets are when seeing this part in the console, especially given that there is no console message at all mentioning the `lst` part, which by the way is quite slow.

Reply:
Good point. Also added a comment explaining a bit more what's built and why.